Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -2478,6 +2335,12 @@ public async Task DoNotDeleteCanonicalChainWhenBlockDownloadFailed()
             return task;
         }
 
+        private Task StopAsync<T>(Swarm<T> swarm)
+            where T : IAction, new()
+        {
+            return swarm.StopAsync(TimeSpan.FromMilliseconds(10));
```

Comment:
Is it okay if we give it such a short timeout?

Reply:
I tested several timeout parameters like 500, 50, 10, 1. The tests failed only if the timeout was 1ms.